libnmstate/nm/context.py

```diff
@@ -208,13 +212,12 @@ def wait_all_finish(self):
 
         if self._error:
             # The queue and error should be flush and perpare for another run
+            self._cancellable.cancel()
+            self.refresh_content()
```

Comment:
After this patch I will investigate if `refresh_content()` is still needed.

Reply:
That is trackked at https://github.com/nmstate/nmstate/issues/1398